src/main/webapp/app/modules/canvas/filter/filter-element.tsx

```diff
@@ -172,11 +171,12 @@ const FilterElement = (props: IFilterElementProp) => {
           ) : (
             <View>
               <AsyncSelect
-                cacheOptions
                 value={defaultValues}
                 isMulti
                 isSearchable={true}
+                closeMenuOnSelect={false}
                 classNamePrefix="select"
+                cacheOptions
```

Comment:
why you are caching an options. they must be loaded when you take the cursor on it